### PR TITLE
 To do app React version

### PR DESCRIPTION
**Pull request by mohisa302**
###  To do app React version
in this project, I create to-do app with React tutorial:

- Added local storage functionality 
- Added Router to multiply pages
- Added edit and delete tasks option

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import TodoApp from './TodoApp';
+
+function App() {
+  return (
+    <div>
+      <TodoApp />
+    </div>
```
Comment (Ouail-01) [inline src/components/App.js]: [OPTIONAL] Please replace unnecessary root div elements with React fragments

File: src/components/TodoItem.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  }
+
+  return (
+    <li className={styles.item}>
+      <div className={styles.content} style={viewMode}>
+        <input
+          type="checkbox"
+          checked={itemProp.completed}
+          onChange={() => { handleChange(itemProp.id); }}
+        />
```
Comment (Ouail-01) [inline src/components/TodoItem.js:56]: Please update the style when the item is checked, just like in the tutorial.
Yours 

Screenshot 2023-02-24 at 14 51 30


In the tutorial
Screenshot 2023-02-24 at 14 50 41

**Review by Ouail-01 — CHANGES_REQUESTED** [review]
Hi @mohisa302 , 

You did extremely great on this project! 👏 👏 
However there are a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦   🏃 




## Highlights 👍  
- CRUD feature is functioning as intended!
- Tasks are updated in local storage accordingly when changes are made!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly check the inline comments marked **[OPTIONAL]** under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------

File: src/components/TodoItem.js (hunk 2 above)
Comment (mohisa302) [reply src/components/TodoItem.js:56]: Hello @Ouail-01 , Thanks for your review, I implemented all the requested changes and will be appreciate if you review my code again, thanks.

**Comment by Ouail-01** [comment]
Hi 👋 @mohisa302 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✔️ The PR title and description look good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ouail-01 — APPROVED** [review]
Hi 👋 @mohisa302 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✔️ The PR title and description look good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**